xonsh/built_ins.py

```diff
@@ -351,8 +351,39 @@ def run_subproc(cmds, captured=True):
             stdin = PIPE
         else:
             stdin = prev_proc.stdout
-        proc = Popen(aliased_cmd, universal_newlines=uninew, env=ENV.detype(),
-                     stdin=stdin, stdout=stdout)
+        try:
+            proc = Popen(aliased_cmd, universal_newlines=uninew, env=ENV.detype(),
+                         stdin=stdin, stdout=stdout)
+        except FileNotFoundError:
+            cmd = aliased_cmd[0]
+            print('xonsh: subprocess mode: command not found: {0}'.format(cmd))
```

Comment:
No problem (I am assuming you mean mostly the suggestion stuff should move).  Am I correct in assuming that tools.py is a fine place to dump these kinds of things?  Or should I make a new suggester.py file or something like that?